toolset/utils/docker_helper.py

```diff
@@ -218,6 +227,10 @@ def watch_container(docker_container, docker_file):
                 remove=True,
                 log_config={'type': None})
 
+#            print('WAITING while you connect to 4005')
+#            import time
+#            time.sleep(15)
+
```

Comment:
Please try to don't mix toolset changes, with frameworks changes in the same PR.

Reply:
The toolset change need to be approved first.
If it isn't approved, your changes to the frameworks will be not merged also.

Reply:
Ok, I'll separate this pull into two.